test cases/common/16 configure file/generator.py

```diff
@@ -1,5 +1,9 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 
+# On some platforms "python" points to Python 2
+# on others to Python 3. Work with both.
+
+from __future__ import print_function
```

Comment:
Don't need to do this import if we'll use the python3 module to run it with python3.